content/site-policy/acceptable-use-policies/github-bullying-and-harassment.md

```diff
@@ -20,4 +20,4 @@ We do not tolerate harassment, bullying, or abuse of any kind, whether directly
 - Disingenuously participating in conversation in a way that instigates conflict or undermines sincere discussion
 - Creating alternative accounts specifically to evade moderation action taken by GitHub staff or users
 
-Please note, not all unwelcome conduct is necessarily considered harassment. For example, disagreeing with another user or downvoting their comments may not rise to the level of harassment on our platform. In addition, sharing criticism of public figures or projects, or topics of public interest, does not necessarily fall under this policy. However, we encourage you to be mindful in how you engage with other users and the platform, as this activity may still violate our restriction on disrupting the experience of other users.
+Please note, not all unwelcome conduct is necessarily considered harassment. For example, disagreeing with another user or down-voting their comments may not rise to the level of harassment on our platform. In addition, sharing criticism of public figures or projects, or topics of public interest, does not necessarily fall under this policy. However, we encourage you to be mindful in how you engage with other users and the platform, as this activity may still violate our restriction on disrupting the experience of other users.
```

Comment:
ℹ️ Same here; if there's a preference for the non-hyphenated version this should be a quick revert.